bundles/org.openhab.binding.volvooncall/example.events.txt

```diff
@@ -0,0 +1,73 @@
+# Some events from openhab to see how the api responds over different charging events during a cycle.
```

Comment:
I don't think this file should be kept. If it's an helper for the end user, it could be integrated in the README.md

Reply:
It's really notes for developers, so I thought leaving it in the root folder would be helpful.

Reply:
I think you should move it to a doc folder